package.nix

```diff
@@ -30,15 +29,13 @@ rustPlatform.buildRustPackage {
   cargoLock.lockFile = ./Cargo.lock;
   nativeBuildInputs = [
     nix
-    rustfmt
     clippy
     makeWrapper
   ];
   env.NIX_CHECK_BY_NAME_EXPR_PATH = "${runtimeExprPath}";
   env.NIX_PATH = "test-nixpkgs=${testNixpkgsPath}:test-nixpkgs/lib=${nixpkgsLibPath}";
   preCheck = initNix;
   postCheck = ''
-    cargo fmt --check
```

Comment:
Can we use `treefmt --fail-on-change` here?

Reply:
This is now handled by https://github.com/NixOS/nixpkgs-check-by-name/blob/ff95d9b072e1726cf2b88a3e9e1314df32df373b/default.nix#L75-L81 (just added the comment above to make this clearer)